### PR TITLE
[Tables] Regen with latest autorest

### PR DESCRIPTION
**Pull request by YalinLi0312**

### CODE REVIEW

**Review by iscai-msft — APPROVED** [review]

File: sdk/tables/azure-data-tables/azure/data/tables/_generated/_configuration.py [hunk 1]
```diff
@@ -6,54 +6,56 @@
...
+
         if url is None:
             raise ValueError("Parameter 'url' must not be None.")
-        super(AzureTableConfiguration, self).__init__(**kwargs)
 
         self.url = url
-        self.version = "2019-02-02"
-        kwargs.setdefault('sdk_moniker', 'data-tables/{}'.format(VERSION))
+        self.version = version
+        kwargs.setdefault("sdk_moniker", "table/{}".format(VERSION))
```
Comment (annatisch) [inline sdk/tables/azure-data-tables/azure/data/tables/_generated/_configuration.py:46]: Looks like me might need to correct the user-agent format.

File: sdk/tables/azure-data-tables/azure/data/tables/_table_client.py [hunk 2]
```diff
@@ -175,7 +174,7 @@ def get_table_access_policy(
         """
         timeout = kwargs.pop("timeout", None)
         try:
-            _, identifiers = self._client.table.get_access_policy(
+            identifier = self._client.table.get_access_policy(
```
Comment (annatisch) [inline sdk/tables/azure-data-tables/azure/data/tables/_table_client.py:177]: Why did this output format change?

File: sdk/tables/azure-data-tables/azure/data/tables/_generated/_configuration.py (hunk 1 above)
Comment (annatisch) [reply sdk/tables/azure-data-tables/azure/data/tables/_generated/_configuration.py:46]: Looks like this is being set elsewhere:
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/tables/azure-data-tables/azure/data/tables/_base_client.py#L239
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/tables/azure-data-tables/azure/data/tables/_sdk_moniker.py#L11

File: sdk/tables/azure-data-tables/azure/data/tables/_generated/operations/_operations.py [hunk 3]
```diff
@@ -0,0 +1,2242 @@
...
+
+        if cls:
+            return cls(pipeline_response, deserialized, response_headers)
+
+        return deserialized
+
+    @distributed_trace
+    def get_access_policy(
+        self, table: str, *, timeout: Optional[int] = None, request_id_parameter: Optional[str] = None, **kwargs: Any
+    ) -> _models.SignedIdentifier:
```
Comment (YalinLi0312) [inline sdk/tables/azure-data-tables/azure/data/tables/_generated/operations/_operations.py:1818]: @iscai-msft The return is actually a list of `SignedIdentifier`, can you help to fix it?

**Pull request merged**